docs/source/configuration.rst

```diff
@@ -99,4 +99,52 @@ RESPONSE_VALIDATION
     that they conform to the protocol. This should only be used for development
     purposes.
 
+OIDC_PROVIDER
```

Comment:
@jeromekelleher do you think any of these new config vars should be added to the html display?